tests/src/system/basic/WskActionSequenceTests.scala

```diff
@@ -92,11 +92,31 @@ class WskActionSequenceTests
     it should "create, and get an action sequence" in withAssetCleaner(wskprops) {
         (wp, assetHelper) =>
             val name = "actionSeq"
-            val artifacts = "/whisk.system/watson/speechToText,/whisk.system/watson/translate"
+            val packageName = "watson"
+            val speechToTextName = s"$packageName/speechToText"
+            val transalteName = s"$packageName/translate"
+            val fullSpeechToTextName = s"/guest/$speechToTextName"
+            val fullTranslateName = s"/guest/$transalteName"
+
+            assetHelper.withCleaner(wsk.pkg, packageName) {
+                (pkg, _) => pkg.create(packageName, shared = Some(true))(wp)
+            }
+
+            assetHelper.withCleaner(wsk.action, speechToTextName) {
+                val file = Some(TestUtils.getTestActionFilename("speechToText.js"))
+                (action, _) => action.create(speechToTextName, file, shared = Some(true))(wp)
+            }
+
+            assetHelper.withCleaner(wsk.action, transalteName) {
+                val file = Some(TestUtils.getTestActionFilename("translate.js"))
+                (action, _) => action.create(transalteName, file, shared = Some(true))(wp)
+            }
```

Comment:
Can we come up with a different sequence of action to not having this huge dependency on Watson services and so on in the main repository?